testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/federation/ldap/LDAPProvidersIntegrationTest.java

```diff
@@ -106,6 +106,7 @@ protected void afterImportTestRealm() {
         testingClient.server().run(session -> {
             LDAPTestContext ctx = LDAPTestContext.init(session);
             RealmModel appRealm = ctx.getRealm();
+            session.getContext().setRealm(appRealm);
```

Comment:
Would be possible to set the realm on the `org.keycloak.testsuite.rest.TestingResourceProvider#runOnServer` instead?